src/agent/agent.service.ts

```diff
@@ -76,4 +77,23 @@ export class AgentService {
         };
         return await this.agentCaller.callAgent(process.env.AGENT_ID, process.env.ADMIN_API_KEY, 'POST', 'wallet/did/public', params);
     }
+
+    /*
```

Comment:
Could you make this a javadoc instead? It would help with auto-completion for some IDEs. I.e. `/**`